manager.cpp

```diff
@@ -272,8 +276,104 @@ void Manager::updateObjects(
     }
 }
 
+void Manager::clearPersistency(const std::vector<std::string>& paths)
+{
+    for (const auto& objPath : paths)
+    {
+        std::string absObjPath(std::string(PIM_PERSIST_PATH) +
+                               "/xyz/openbmc_project/inventory");
+        absObjPath.append(objPath);
+
+        for (const auto& file : directory_iterator(absObjPath))
+        {
+            if (!remove(file.path()))
```

Comment:
can't file also be a directory?  or does remove just fail on a directory.  If so, better to check that it's a normal file first.

Reply:
ack

Reply:
Line 290: 
    bool std::filesystem::remove( const std::filesystem::path& p ); returns true on success

whereas,

Line 297: 
    int std::remove( const char* fname );    returns 0 on success